package/MDAnalysis/analysis/dssp/dssp.py

```diff
@@ -319,7 +319,7 @@ def __init__(
                 ):
                     raise ValueError(
                         (
-                            "Residue {calpha.residue} contains wrong number of hydrogens: "
+                            f"Residue {calpha.residue} contains wrong number of hydrogens: "
```

Comment:
Is this covered by a test? I.e. if it is, why did the error message matching not fail? If it isn't, then we should cover it by a test.

Reply:
@IAlibay fixed, thanks.